taxonomy-editor-frontend/src/pages/editentry/LanguageSelectionDialog.tsx

```diff
@@ -40,29 +43,27 @@ const LanguageSelectionDialog = ({
             id="multiple-lang-checkbox"
             multiple
             value={newShownLanguageCodes}
-            onChange={handleLanguageCheckToggle}
+            onChange={(event) =>
+              setNewShownLanguageCodes(event.target.value as string[])
```

Comment:
Is `string[]` required here?

Reply:
it is required, this is due to the `multiple` prop. Added a note